auter.yumdnfModule

```diff
@@ -73,14 +73,14 @@ function apply_updates() {
       # Save the staged rpms to a variable
       RPMS="${DOWNLOADDIR}/${CONFIGSET}/*.rpm"
       # Check if the updates staged are still required
-      "${PACKAGE_MANAGER}" check-update $(rpm -qp ${RPMS}) &>/dev/null
+      "${PACKAGE_MANAGER}" check-update $(rpm -qp ${RPMS}) &>${DATADIR}/last-apply-output-${CONFIGSET}
```

Comment:
No longer relevant after proposed PR to switch to rpm for checking updates validity.